grakn-core/src/main/java/ai/grakn/graql/analytics/ConnectedComponentQuery.java

```diff
@@ -44,9 +44,10 @@
      * Return the instances in each cluster after executing the query. By default, the size of each cluster is
      * returned after executing the query.
      *
+     * @param isMembersSet whether this query will return the members of each cluster
      * @return a ConnectedComponentQuery with members flag set
      */
-    ConnectedComponentQuery<Map<String, Set<String>>> members();
+    ConnectedComponentQuery<Map<String, Set<String>>> members(boolean isMembersSet);
```

Comment:
Awkwardly, the return type of this is wrong now if you do `query.members(false)` :(

You could do this:
```
<T> ConnectedComponentQuery<T> members(boolean isMembersSet);
```
which is OK, but it mean the caller can specify any `<T>` they want, even if it's wrong.

Alternatively, this is more correct but ugly:
```
ConnectedComponentQuery<Map<String, Set<String>>> membersOn();
ConnectedComponentQuery<Map<String, Long>> membersOff();
```

I don't know what the best choice is!